### ISSUE TITLE
[Suggestion] Cumulative counting with ROI

### ISSUE THREAD

**Issue by aparico** [theAIGuysCode/yolov4-deepsort#39]
Thanks for your great repo!

But it would be nice if you have a **cumulative counting mode** to count objects going through some _ROI line_. 

Examples: 

[https://github.com/ahmetozlu/tensorflow_object_counting_api#cumulative-counting-mode-tensorflow-implementation](url)

[https://github.com/molimat/YOLOv4-Counter-in-TF#outputs](url)

It would be greatly appreciated!

**Comment by Timbimjim** [comment]
Yes I am also looking for smt like this. I already tested your counting project, and it works really nicely.

Would it be able to combine the counting with this repo?

Thx for the great work so far!

Have a nice day mate

**Comment by xanthan011** [comment]
hello people, this code provides the coordinates of the bounding boxes of the object detected. I suggest u can take the center of the 4 points and write a code that counts when object is passed through that virtual line. Its simple in python. Consider the image a 2-D plane and it will be a simple mathematical problem to be implemented in python
